examples/client/object_server.c

```diff
@@ -423,111 +356,13 @@ static uint8_t prv_server_discover(lwm2m_context_t *contextP,
 
 #ifndef LWM2M_VERSION_1_0
         /* Remove optional resources that don't exist */
```

Comment:
Would use this this as a starting point to document the newly created function, not "waste" it here.